.github/workflows/prod.yaml

```diff
@@ -30,6 +30,9 @@ jobs:
     needs: release
     environment: prod
     steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
we use `fetch-depth: 0` everywhere (meaning checkout all history), but I think it might only be needed for `release`? i'll leave this here for now, but perhaps we should check all of these out (pun intended).

Reply:
It is need for deploy targets as well as it creates version.json

Reply:
I think could do without on ci maybe. Not sure. Will try on tractor

Reply:
It is needed for ci, because that does a build and it uses `$(VERSION)` in a linker flag which is based on the tags which need history.